src/index.js

```diff
@@ -61,7 +95,7 @@ function start() {
     title: 'Headset',
     maximizable: false,
     useContentSize: true,
-    icon: path.join(__dirname, 'icons', 'Headset.ico'),
+    icon: windowIcon,
     webPreferences: { nodeIntegration: true },
```

Comment:
One option is missing for darwin: `titleBarStyle`
We can add it like this:
```
let titleBarStyle = 'default';

...

if (OS === 'darwin') {
  ...
  titleBarStyle = 'hiddenInset';
}

...

win = new BrowserWindow({
  ...
  titleBarStyle
});
```

Reply:
I forgot to add that option. There's no need to test for the OS, that option only works on macOS and has no effect on the others